gamemodes/core/user-interface/ui_peds.inc

```diff
@@ -67,6 +67,9 @@ hook OnGameModeInit() {
 
 public OnPlayerStartRobbery(playerid, actorid, bool:robbed_recently)
 {
+    if(Player_GetClass(playerid) != TEAM_CIVILIAN) {
```

Comment:
OnPlayerRequestRobbery is made, this is deprecated.

Reply:
Done